README.md

```diff
@@ -38,15 +41,21 @@ The SDK supports Python 2 ≥ 2.7 and Python 3 ≥ 3.4. Only the official CPytho
 from <https://python.org>) is supported and only on Linux (musl libc is currently not supported) and Windows with the x86 (including
 x86-64) architecture. Additionally, `pip` ≥ 8.1.0 (2016-03-05) is required for installation.
 
-The Dynatrace OneAgent SDK for Python is a wrapper of the [Dynatrace OneAgent SDK for C/C++](https://github.com/Dynatrace/OneAgent-SDK-for-C) and therefore the SDK for C/C++ is required and delivered with the Python SDK. See [here](https://github.com/Dynatrace/OneAgent-SDK-for-C#dynatrace-oneagent-sdk-for-cc-requirements) for its requirements, which also apply to the SDK for Python.
+The Dynatrace OneAgent SDK for Python is a wrapper of the [Dynatrace OneAgent SDK for C/C++](https://github.com/Dynatrace/OneAgent-SDK-for-C)
+and therefore the SDK for C/C++ is required and delivered with the Python SDK. See
+[here](https://github.com/Dynatrace/OneAgent-SDK-for-C#dynatrace-oneagent-sdk-for-cc-requirements)
+for its requirements, which also apply to the SDK for Python.
 
-The version of the SDK for C/C++ that is included in each version of the SDK for Python is shown in the following table along with the required Dynatrace OneAgent version (it is the same as [listed in the OneAgent SDK for C/C++'s documentation](https://github.com/Dynatrace/OneAgent-SDK-for-C/blob/master/README.md#compatibility-of-dynatrace-oneagent-sdk-for-cc-releases-with-oneagent-releases)).
+The version of the SDK for C/C++ that is included in each version of the SDK for Python is shown in the following table along with the required
+Dynatrace OneAgent version (it is the same as
+[listed in the OneAgent SDK for C/C++'s documentation](https://github.com/Dynatrace/OneAgent-SDK-for-C/blob/master/README.md#compatibility-of-dynatrace-oneagent-sdk-for-cc-releases-with-oneagent-releases)).
 
 <a name="pycversiontab"></a>
 
 |OneAgent SDK for Python|OneAgent SDK for C/C++|Dynatrace OneAgent|
 |:----------------------|:---------------------|:-----------------|
 |1.0                    |1.1.0                 |≥1.141            |
+|1.1                    |1.3.1                 |≥1.151            |
```

Comment:
i suggest to add link to release notes / upgrade path for 1.1

Reply:
see .NET SDK release notes for inspiration (also API change): https://github.com/Dynatrace/OneAgent-SDK-for-dotnet/releases

Reply:
fixed

Reply:
it's already documented in the drafted release notes, too.